semgrep/semgrep/output.py

```diff
@@ -62,8 +62,8 @@ def _build_time_target_json(
 
     target_json["path"] = path_str
     target_json["num_bytes"] = num_bytes
-    timings = [profiling_data.get_run_times(rule.id, path_str) for rule in rules]
-    target_json["parse_times"] = [timing.parse_time for timing in timings]
+    target_json["parse_time"] = profiling_data.get_file_parse_time(target)
```

Comment:
Changed `parse_times` in the json time output to just report a single time instead of a list of times. Does this affect our benchmarking stuff @emjin @colleend?

Reply:
This is fine for the benchmarking I did (I don't use -json-time), but @colleend might have changed that

Reply:
no worries, benchmarking still doesn't use -json-time 👍